lib/command/run.rb

```diff
@@ -136,10 +196,52 @@ def runner_script # rubocop:disable Metrics/MethodLength
       script
     end
 
-    def run_in_replica
+    def non_interactive_runner_script
```

Comment:
The code here is essentially the same that was in `run:detached` for the runner script.